uobtheatre/bookings/test/factories.py

```diff
@@ -58,6 +58,8 @@ class TicketFactory(factory.django.DjangoModelFactory):
         "uobtheatre.discounts.test.factories.ConcessionTypeFactory"
     )
 
+    checked_in_by = factory.SubFactory(UserFactory)
+
```

Comment:
By default this will mean any ticket we create via a factory is partially checked in (because they have a `checked_in_by` but not a `checked_in_at`. We either need to set both or neither to comply with our logic. I would side on default to not having tickets checked in from a factory, to I would remove this line. (I realise this means you have to provide one when you create in your tests - @JElgar might be able to chime in about if we can have optional states for our factories that would do this for you)